reference/env_vars.rst

```diff
@@ -76,9 +76,11 @@ instead of Yellow, Cyan, White.
 CONAN_COLOR_DISPLAY
 -------------------
 
-**Defaulted to**: ``True``/``1``
+**Defaulted to**: Not defined
```

Comment:
Mention that by default (not defined) it will use color display when tty is detected.